net/usrsock/usrsock_ioctl.c

```diff
@@ -233,6 +234,11 @@ int usrsock_ioctl(FAR struct socket *psock, int cmd, FAR void *arg,
 errout_unlock:
   net_unlock();
 
+  if (ret == OK && cmd == FIONBIO)
```

Comment:
Since the usrsock server always put the socket in nonblocking mode, should we return -ENOTTY at the beginning?

Reply:
Done